src/ui_contextmenu.cpp

```diff
@@ -293,7 +300,79 @@ class OpenLyricsContextSubItem : public contextmenu_item_simple
                                                  core_api::get_main_window(),
                                                  "Marking tracks as instrumental...");
             } break;
+            case cmd_seek_to_next_lyric: {
+                auto& panels = get_active_panels();
+                if (!panels.empty())
+                {
+                    LyricPanel* panel = panels[0];
+                    LyricData lyrics = panel->get_lyrics();

```

Comment:
We should try to avoid copying the entire `LyricData` here.

```suggestion
                    const LyricData& lyrics = panel->get_lyrics();
```